release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md

```diff
@@ -1,4 +1,65 @@
-<!-- Initial implementation of the Bevy Remote Protocol (Adopted) -->
-<!-- https://github.com/bevyengine/bevy/pull/14880 -->
+We have introduced the Bevy Remote Protocol, which allows the ECS of a running
+Bevy application to be interacted with remotely. This can be used, for example,
+to inspect and edit entities and their components at runtime. We anticipate 
+that this will be used primarily to create things like inspectors for editing
```

Comment:
I think that calling out the multi-process editor architecture is helpful here. Lots of hedging on "experimental" please.